example/tests/requestMocking.test.js

```diff
@@ -0,0 +1,80 @@
+import { RequestMocker, Element, Helpers } from "test-juggler";
+const fs = require("fs");
+
+describe("Request mocking by recording and replaying specific requests", () => {
+
+    let mocker = new RequestMocker();
+    let helpers = new Helpers();
+
+    it("Recording all requests and replaying it on the second test run google", async () => {
+        //Arrange
+        await mocker.start();
+
+        //Act
+        await helpers.goToUrlAndLoad("https://downforeveryoneorjustme.com/google");
+        await mocker.stop();
+
+        //Assert
+        expect(fs.existsSync("mockData\\Request mocking by recording and replaying and indercetipt specific requests\\Recording all requests and replaying it on the second test run google\\interceptors\\get.json")).toBe(true);
```

Comment:
Path should be updated according to describe: "Request mocking by recording and replaying and indercetipt specific requests" > "Request mocking by recording and replaying specific requests"